src/views/home/components/bridge-form/bridge-form.view.tsx

```diff
@@ -96,99 +78,39 @@ const BridgeForm: FC<BridgeFormProps> = ({
   };
 
   const onTokenDropdownClick = () => {
-    setFilteredTokens(tokens);
+    setIsTokenListOpen(true);
   };
 
-  const onTokenListTokenSelected = (token: Token) => {
+  const handleSelectToken = (token: Token) => {
     setToken(token);
-    setFilteredTokens(undefined);
+    setIsTokenListOpen(false);
     setAmount(undefined);
   };
 
-  const updateTokenList = (tokensWithBalance: Token[], tokenListSearchTerm: string) => {
-    const filteredTokens = tokensWithBalance.filter(getTokenFilterByTerm(tokenListSearchTerm));
-    setFilteredTokens(filteredTokens);
-    setTokenListCustomToken({ status: "pending" });
-    if (
-      selectedChains &&
-      ethersUtils.isAddress(tokenListSearchTerm) &&
-      filteredTokens.length === 0
-    ) {
-      setTokenListCustomToken({ status: "loading" });
-      const setBalanceAndListCustomToken = (token: Token) => {
-        getTokenBalance(token, selectedChains.from)
-          .then((balance) => {
-            callIfMounted(() => {
-              setFilteredTokens([{ ...token, balance }]);
-              setTokenListCustomToken({ status: "successful", data: { ...token, balance } });
-            });
-          })
-          .catch(() => {
-            callIfMounted(() => {
-              setFilteredTokens([{ ...token, balance: undefined }]);
-            });
-          });
-      };
-      void getTokenFromAddress({
-        address: tokenListSearchTerm,
-        chain: selectedChains.from,
-      })
-        .then(setBalanceAndListCustomToken)
-        .catch(() =>
-          getTokenFromAddress({
-            address: tokenListSearchTerm,
-            chain: selectedChains.to,
-          })
-            .then(setBalanceAndListCustomToken)
-            .catch(() =>
-              callIfMounted(() => {
-                setTokenListCustomToken({
-                  status: "failed",
-                  error:
-                    "The token contract cannot be called on any network. Please make sure the contract is deployed on a network.",
-                });
-              })
-            )
-        );
-    }
-  };
-
-  const onTokenListClosed = () => {
-    setFilteredTokens(undefined);
-    setTokenListCustomToken({ status: "pending" });
-    setTokenListSearchInputValue("");
+  const handleCloseTokenSelector = () => {
```

Comment:
Mmmm good catch! This was a convention we had in the `wallet-ui` and in the `batch-explorer` (you can check it that the `AmountInput` component is still using it). Not sure why we lost it here as it's very descriptive and I really like it. But we are not following it, so I've changed all the `handle...` to `on...`.

This convention is very useful because, for example, it prevents collisions of this type:

<img width="344" alt="Captura de pantalla 2022-11-02 a las 20 40 49" src="https://user-images.githubusercontent.com/6737805/199587471-a7d3bdc3-cde6-4c65-9dd9-10538a727478.png">

If you rename `handleAddToken` to `onAddToken` will create a collision with the listener being passed from the parent component, so you need to workaround the name to something like `onAddTokenToList` to prevent the collision.

Reply:
I'm ok with both prefixes, leave it as you prefer.